### PR TITLE
Add Inventory modules to composer.json

### PR DESCRIPTION
**Pull request by mslabko**
### Fixed Issues (if relevant)

1. magento/ece-tools#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: src/Command/Dev/UpdateComposer/ComposerGenerator.php [hunk 1]
```diff
@@ -244,6 +244,9 @@ private function addModules(array $repoOptions, array $composer): array
                     $add($dir);
                 }
             }
+            foreach (glob($baseRepoFolder . '/Inventory*') as $dir) {
```
Comment (shiftedreality) [inline src/Command/Dev/UpdateComposer/ComposerGenerator.php:247]: Please add suppress warning

**Review by shiftedreality — CHANGES_REQUESTED** [review]

File: src/Command/Dev/UpdateComposer/ComposerGenerator.php (hunk 1 above)
Comment (oshmyheliuk) [reply src/Command/Dev/UpdateComposer/ComposerGenerator.php:247]: Also, we have another package that has the same structure `security-package`. And later we can have other similar packages. 
I think it maybe better to do it more robust.
For example
```
        Inventory:
          branch: 1.0-develop
          repo: https://github.com/magento/Inventory.git
          type: flat-structure
```
And if type == `flat-structure` we will automatically look for modules in the root directory.
@YPyltiai if you agree I can create a task for that and close this PR.

Comment (YPyltiai) [reply src/Command/Dev/UpdateComposer/ComposerGenerator.php:247]: Yeah, it sounds good for me, @oshmyheliuk .

**Comment by oshmyheliuk** [comment]
@mslabko Thank you for pr.
Will be fixed in this task https://jira.corp.magento.com/browse/MCLOUD-5665

**Pull request closed without merging**